config/initializers/config.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+Config.setup do |config|
+  config.const_name = 'Settings'
+  config.use_env = true
+  config.env_prefix = 'SETTINGS'
+  config.env_separator = '__'
+  # config.env_converter = :downcase
```

Comment:
Cruft?

Reply:
Not really.  This is something we usually do, but we have to convert all the Settings variables to lowercase first. Not something I want to add to this pr.